rmt-solution.js

```diff
@@ -0,0 +1,138 @@
+const VISITED = '*';
+const DESK = '-';
+const WALL = 'x';
+const DESTINATION = 'C';
+
+
+class Node {
+  constructor(name, children) {
```

Comment:
This was my mistake, a better name is `neighbors`. Children are for trees, neighbors are for graphs.